### ISSUE TITLE
React Date Picker with moment

### ISSUE THREAD

**Issue by davidkhanpk** [Jerga99/portfolio-jerga#7]
react date picker module after version 2.x now only working with javascript date functions. After 2.x it is not supporting moment date functions.

**Comment by davegahn** [comment]
You can use the any library with same functionality.
This https://github.com/date-fns/date-fns worked for me.

**Comment by davidkhanpk** [comment]
Yes, I am using it to parse string to ISO and date format. Because react-dateppicker no longer accept moment and giving "Invalid time value" errors.

**Comment by davegahn** [comment]
I refused to use react-datepicker at all. You can use a simple date field of reactstrap, or find alternatives.
In my case I use material-ui that contains a date picker.
